airflow/contrib/operators/field_validator.py

```diff
@@ -0,0 +1,168 @@
+from airflow import AirflowException, LoggingMixin
```

Comment:
Can you add a license and high-level docstring for this file? It seems like it should go in a contrib utils folder rather than operators.

I'm not too familiar with the context for why this file was introduced.

Reply:
Indeed, that's a goal to have it reusable for all operators. I usually implement those kind of utils in the way that I first make it part of one function, then when it is reused for another  I move it  to common utils and only when you use it for the third time, it becomes truly reusable :). But indeed in this case it's already pretty generic, so I am happy to move it to utils.